net-im/discord/discord-0.0.58-r1.ebuild

```diff
@@ -15,7 +15,7 @@ CHROMIUM_LANGS="
 inherit chromium-2 desktop linux-info optfeature unpacker xdg
 
 DESCRIPTION="All-in-one voice and text chat for gamers"
-HOMEPAGE="https://discordapp.com"
+HOMEPAGE="https://discord.com/"
```

Comment:
nit: can we remove the final '/'

Reply:
Usually a (complete) HTTP URL should have a scheme, a host and a path, hence the `/` for the path which is the default value when missing.
And also, this is the feedback returned by `pkgcheck scan --net -- net-im/discord/`:
```
net-im/discord
  RedirectedUrl: version 0.0.58-r1: HOMEPAGE: permanently redirected: https://discordapp.com -> https://discord.com/
```

Reply:
ok i dont have a strong opinion so its fine as is